### PR TITLE
[CB-115][HAWTHORN] Fix "Expand All" button language is changed after clicking on it

### PR DESCRIPTION
**Pull request by 0lha**
**Description:** Fix "Expand All" button language is changed after clicking on it; fix ru translation

**Youtrack:** https://youtrack.raccoongang.com/issue/CB-115

### CODE REVIEW

**Comment by raccoondeploy** [comment]
Build finished.

**Comment by raccoondeploy** [comment]
Refer to this link for build results (access rights to CI server needed): 

<a href='http://jenkins-ci.raccoongang.com/job/hawthorn-python-unittests-pr/28/'>http://jenkins-ci.raccoongang.com/job/hawthorn-python-unittests-pr/28/</a>
<a href='http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/435/'>http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/435/</a>
<a href='http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/436/'>http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/436/</a>
<a href='http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/437/'>http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/437/</a>
<a href='http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/439/'>http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/439/</a>
<a href='http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/440/'>http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/440/</a>
<a href='http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/441/'>http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/441/</a>
<a href='http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/442/'>http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/442/</a>
<a href='http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/443/'>http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/443/</a>
<a href='http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/444/'>http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/444/</a>
<a href='http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/445/'>http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/445/</a>
<a href='http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/446/'>http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/446/</a>
<a href='http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/447/'>http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/447/</a>
<a href='http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/448/'>http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/448/</a>
<a href='http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/450/'>http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/450/</a>
<a href='http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/449/'>http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/449/</a>

File: openedx/features/course_experience/static/course_experience/js/CourseOutline.js [hunk 1]
```diff
@@ -82,12 +82,12 @@ export class CourseOutline {  // eslint-disable-line import/prefer-default-expor
...
         sectionAction = collapseSection;
         toggleAllSpan.classList.add(extraPaddingClass);
-        toggleAllSpan.innerText = 'Expand All';
+        toggleAllSpan.innerText = gettext('Expand All');
       } else {
         toggleAllButton.setAttribute('aria-expanded', 'true');
         sectionAction = expandSection;
         toggleAllSpan.classList.remove(extraPaddingClass);
-        toggleAllSpan.innerText = 'Collapse All';
+        toggleAllSpan.innerText = gettext('Collapse All');
```
Comment (idegtiarov) [inline openedx/features/course_experience/static/course_experience/js/CourseOutline.js]: What about `Collapse All`? Does it have the translation?

Comment (0lha) [reply openedx/features/course_experience/static/course_experience/js/CourseOutline.js]: Yes, it has https://github.com/0lha/edx-platform/blob/ginkgo-rg/conf/locale/ru/LC_MESSAGES/djangojs.po#L7276

There was ru translations for "Expand All"/"Collapse All" in djangojs.po, but translation for django.po was missed, so button haven't been translated until click.
I have another thought. Should I check the same situation for other languages and transfer data from djangojs.po to django.po if necessary?

Comment (idegtiarov) [reply openedx/features/course_experience/static/course_experience/js/CourseOutline.js]: got it, thanks

**Comment by oksana-slu** [comment]
не актуально, в гитлабе пофикшено

**Pull request closed without merging**